pwndbg/aglib/stack.py

```diff
@@ -120,26 +127,50 @@ def _fetch_via_exploration() -> Dict[int, pwndbg.lib.memory.Page]:
     An alternative to this is dumping this functionality completely and this
     will be decided hopefully after a next release.
     """
-    stacks: Dict[int, pwndbg.lib.memory.Page] = {}
+    if auto_explore.value == "warn":
+        print(
+            M.warn(
+                "Warning: All methods to detect STACK have failed.\n"
+                "You can explore STACK using exploration, but it may be very slow.\n"
+                "To explicitly explore, use the command: `stack_explore`\n"
+                "Alternatively, enable it by default with: `set auto-explore-stack yes`"
```

Comment:
Isn't this a bit weird that `fetch_via_exploration` does print this warning? Seems like maybe the code that uses it should do that?

Unless there are many places for it? I don't know huh.